mp/src/game/shared/momentum/mom_stickybomb.cpp

```diff
@@ -0,0 +1,447 @@
+#include "cbase.h"
+#include "mom_stickybomb.h"
+#include "weapon/weapon_mom_stickybomblauncher.h"
+
+#ifndef CLIENT_DLL
+#include "Sprite.h"
+#include "explode.h"
+#include "momentum/mom_triggers.h"
+#endif
+
+#include "tier0/memdbgon.h"
+
+#define MOM_STICKYBOMB_RADIUS 146.0f
+#define MOM_STICKYBOMB_INITIAL_SPEED 900.0f
+#define MOM_STICKYBOMB_MAX_SPEED 2400.0f
+#define MOM_STICKYBOMB_GRAVITY 1.0f // 0.5f in TF2 source code but that doesn't produce the same effect
+#define MOM_STICKYBOMB_FRICTION 0.8f
+#define MOM_STICKYBOMB_ELASTICITY 0.45f
+
+#ifndef CLIENT_DLL
+
+BEGIN_DATADESC(CMomStickybomb)
+// Fields
+DEFINE_FIELD(m_hOwner, FIELD_EHANDLE), DEFINE_FIELD(m_hRocketTrail, FIELD_EHANDLE),
```

Comment:
Spacing here, make each on their own line. Clang formatter doesn't like these but it's way more readable that way